src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./Introduce.scss";
+import { useNavigate } from "react-router-dom";
+
+const IntroduceName = () => {
```

Comment:
```suggestion
const Introduce = () => {
```
파일의 이름과 동일하게 컴포넌트 이름을 구현해주세요!

Reply:
넵!